### DISCUSSION TITLE
get a small loss but the results are completely not correct

### DISCUSSION THREAD

**Discussion by weiroy** [lululxvi/deepxde#1583] [Q&A]
Hi! @lululxvi Thanks very much for your amazing work!
But I encountered a problem. Specifically, the loss is very small, but the results are completely not correct. I am wondering why this happend.
![image](https://github.com/lululxvi/deepxde/assets/152618295/ec98d2c8-de17-47d3-bc59-54dbfb845a5b)
The loss is about 1e-5, in figure 2, the black line is the reference solution and the red line the prediction results of deepxde.
And the governing equations, boundary conditions are as follows:
![1f83bbef4af1624445515c1fc3096f6](https://github.com/lululxvi/deepxde/assets/152618295/36f16e2d-18d1-4194-97e9-1d9d8e5a04f4)
Looking forward to your reply.

**Comment by BraveDrXuTF** [comment]
Are you using tf1 backend? I got similar issue. See [here](https://github.com/lululxvi/deepxde/discussions/1585)

**Comment by yuluoxiansen** [comment]
Have you solved it? I have the same problem.

**Comment by lululxvi** [comment]
Is your problem well defined?

**Reply by weiroy** [reply]
It's my mistake. The right ode system is as follows:

![328b91e202e3d185f142964cdec51fe](https://github.com/lululxvi/deepxde/assets/152618295/8093cd22-24b3-4803-8b5f-f9ab7cafa944)

But I encountered another problem. I have non-dimensionalized the ode system. And the coefficient 0.001 can not be converted to the order of 1 by applying some scaling tricks to the ode system. Some times, the loss is large and the results of pinn are not equal to the reference solution. I guess this is because of the coefficient 0.001, but I don't know how to solve this problem.
